src/test/java/com/wikia/webdriver/common/dataprovider/ads/GermanAdsDataProvider.java

```diff
@@ -81,10 +81,10 @@ private GermanAdsDataProvider() {
          "SOI_SITE: wikia; SOI_SUBSITE: gaming; SOI_SUB2SITE: declashofclans; SOI_SUB3SITE: ; " +
          "SOI_CONTENT: content; SOI_WERBUNG: true; SOI_KEYWORDS: mobile,casual,strategy,war,rts,declashofc"},
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "SOI_CONTENT: content; SOI_WERBUNG: true; SOI_KEYWORDS: mobile,casual,strategy,war,rts,declashofc" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)